core/issuers.chainmail

```diff
@@ -1,56 +1,174 @@
-class Assay[Quantity] {
-  type Label = { issuer: Issuer, description };
-  type Amount = { label: Label, quantity: Quantity };
-  
-  getLabel() ::Label;
-  make(allegedQuantity ?Quantity) ::Amount;
-  vouch(amount ?Amount) ::Amount
-  coerce(amountLike ?Amount) ::Amount;
-  quantity(amount ?Amount) ::Quantity;
-  empty() ::Amount;
-  isEmpty(amount ?Amount) ::boolean;
-  includes(leftAmount ?Amount, rightAmount ?Amount) ::boolean;
-  with(leftAmount ?Amount, rightAmount ?Amount) ::Amount;
-  without(leftAmount ?Amount, rightAmount ?Amount) ::Amount;
+/**
+ * 
+ * Holding an Issuer provides the ability to create amounts and empty purses,
+ * but confers no rights. It is also the mechanism used to get exclusive access
+ * to a Purse or Payment, or to burn some or all of the contained rights.
+ */
+interface Issuer {
+  /* Get the label for this Issuer.
+   * 
+   * Labels can be used to manually construct amounts 
+   */
+  getLabel() -> (Object);
+
+  /* Get the Assay for this Issuer. */
+  getAssay() -> (Assay);
+
+  /** Make an Amount that contains the indicated quantity. */
+  makeAmount(quantity:Quantity) -> (Amount);
+
+  /** Make an empty purse associated for this kind of right. */
+  makeEmptyPurse(name:String) -> (Purse);
+
+  /**
+   * Make a new Payment that has exclusive rights to the amount, only if the
+   * Purse had those rights, exclusively or not.
+   */
+  getExclusive(amount:Amount, src:Purse, name:String = 'a payment') -> (Payment);
+
+  /**
+   * Make a new Payment that has exclusive rights to all the contents of the
+   * purse.
+   */
+  getExclusiveAll(src:Purse, name:String = 'a payment') -> (Payment);
+
+  /**
+   * Burn (destroy) amount of the rights in src. If the rights are fungible,
```

Comment:
Burning is indeed as consensual as `getExclusive`, in the same way, and needs no additional text to explain that.

But your parenthetical "(destroy)" would cause confusion. It would be a fine descriptive word if we did not expect to add an API with a method `destroy` with a very distinct non-consensual meaning. We must keep the `burn` and `destroy` concepts far away from each other.